inst/rmarkdown/templates/frontiers_article/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,169 @@
+---
+title: "Article Title"
+runtitle: "Short Title"
+author:
+  - name: First Author
+    affiliation: '1'
+    etal: LASTNAME # First author's last name. 
+  - name: Co-Author
+    affiliation: '2'
+  - name: Co-Author
+    affiliation: '1,3'
+    email: email@uni.edu # Indicates corresponding Author
+    institution: Laboratory X, Institute X, Department X, Organization X
+    street: Street X
+    city: City X
+    state: State XX # only USA, Australia, Canada
+    zip: Zip Code X
+    country: Country X
+affiliation:
+  - id: '1'
+    department: Department X
+    institution: Institution X
+    city: City X
+    state: State XX # only USA, Australia, Canada
+    country: Country X
+  - id: '2' 
+    department: Department X
+    institution: Institution X
+    city: City X
+    state: State XX # only USA, Australia, Canada
+    country: Country X
+  - id: '3' 
+    department: Department X
+    institution: Institution X
+    city: City X
+    state: State XX # only USA, Australia, Canada
+    country: Country X
+output: rticles::frontiers_article
+csl: frontiers.csl
+bibliography: test.bib
+---
+
+\begin{abstract}
+
+Abstract length and content varies depending on article type. Refer to 
+\url{http://www.frontiersin.org/about/AuthorGuidelines} for abstract requirement
+and length according to article type.
+
+%All article types: you may provide up to 8 keywords; at least 5 are mandatory.
+\tiny
+ \keyFont{ \section{Keywords:} Text Text Text Text Text Text Text Text } 
+
+\end{abstract}
+
+
+# Introduction {-}
+Cite fancy references [@Neuro2013]. As demonstrated in @Gene2012, citations can
+also be automatically reference. Multiple references are separated by semicolons
+[@Neuro2013; @Neurobot2013].
+
+# Results {-}
+
+<!-- Comments should be HTML-style comments -->
+
+## Subsection 1 {-}
+
+You can use `R` chunks directly to plot graphs.  
+
+```{r graph, echo = TRUE, message=FALSE, fig.show = "hide"}
+x <- 0:100
+set.seed(999)
+y <- 2 * (x + rnorm(length(x), sd = 3) + 3)
+plot(x, y)
+```
+
+## Subsection 2 {-}
+
+Frontiers requires figures to be submitted individually, in the same order as
+they are referred to in the manuscript. Figures will then be automatically
+embedded at the bottom of the submitted manuscript. Kindly ensure that each
+table and figure is mentioned in the text and in numerical order. Permission
+must be obtained for use of copyrighted material from other sources (including
+the web). Please note that it is compulsory to follow figure instructions.
+Figures which are not according to the guidelines will cause substantial delay
+during the production process.
+
+# Discussion
```

Comment:
I wonder why most section headers are unnumbered, but a few are numbered (by default).

Reply:
That's a good question.  I was mainly working on the previous commit as a template and then just editing.  I can change if necessary.

Reply:
I was just asking. I don't know the journal requirement. If the journal requires unnumbered sections, we can just do it via a global option (`pdf_document(number_sections = FALSE)`), otherwise I won't use `{-}` frequently throughout the example.